R/misc.R

```diff
@@ -859,9 +859,8 @@ check_new_data <- function(req, object, new_data) {
   step_cls <- class(object)[1]
   step_id <- object$id
   cli::cli_abort(
-    "The following required {cli::qty(col_diff)} column{?s} {?is/are} \
-    missing from `new_data` in step '{step_id}': {col_diff}.",
```

Comment:
The newer error chaining we are doing, makes it so we don't really have to mention the step name any more. so i did that small change as well